app/utils/config.example.js

```diff
@@ -0,0 +1,4 @@
+
+module.exports = {
+  GOOGLE_API_KEY : 'REPLACE_ME'
```

Comment:
We could do that. I could just delete the '&key=...' from the script and leave the rest in place so that adding a key will be easy if we want to track usage or eventually exceed the quota.